src/components/VHeaderOld/VLogoButtonOld.vue

```diff
@@ -29,11 +32,11 @@ import { defineComponent } from '@nuxtjs/composition-api'
 import VLogoLoader from '~/components/VLogoLoader/VLogoLoader.vue'
 import VButton from '~/components/VButton.vue'
 
-import OpenverseLogoText from '~/assets/icons/openverse-logo-text.svg?inline'
+import OpenverseLogoText from '~/assets/icons/openverse-logo-text.svg'
```

Comment:
I don't remember if there was a specific reason we are getting rid of `?inline`. But this is leading to the SVG tag being duplicated in both the SVG file and in the HTML.

Reply:
I wrote about the reasons in this issue: https://github.com/WordPress/openverse-frontend/issues/1715.
I haven't noticed the duplication, I'll look into it, thank you!

Reply:
I don't see the `SVG` tag duplication, @dhruvkb:
<img width="442" alt="Screen Shot 2022-10-27 at 9 23 40 PM" src="https://user-images.githubusercontent.com/15233243/198369025-f5473fbd-c0cf-435a-8f01-741f84c43484.png">